pages/best-practices/avo-functions-alongside-existing-tracking.mdx

```diff
@@ -0,0 +1,214 @@
+      Here you perform the actual event tracking, calling the track/log methods of the analytics SDK. Event name and event
+      properties are provided as parameters.
+
+      Example: analytics.track(eventName, eventProperties);  
+    */
+  },
+  setUserProperties: (userId, userProperties) => {
+    /*
+      You can add User Properties to events in the Trackin Plan in Avo. When an Avo Function with attached user properties
+      is called, this callback is triggered. Here you would attach user properties to the currently identified user in your
+      analytics platform.
+
+      Example: analytics.setUserProperties(userId, userProperties);  
+    */
+  },
+  identify: (userId) => {
+    /*
+      You can add the Identify User action to events in the Trackin Plan in Avo. When calling an Avo Function that includes
+      the Identify User action you'll need to provide a user ID. Calling an Avo Function that has the Identify User action
+      will trigger this callback. The main use cases are signup and login. Here you would pass the user ID to the analytics
+      SDK for it to create a new user or attach a session to an existing user.
+
+      Example: analytics.identify(userId);
+    */
+  },
+  unidentify: () => {
+    /*
+      You can add the Unidentify User action to events in the Trackin Plan in Avo When calling an Avo Function that includes
+      the Unidentify User action this callback will be triggered. Here you would call an analytics SDK method to detach subsequent
+      actions from the currently identified user.
+
+      Example: analytics.identify(null);
+    */
+  },
+};
+```
+
+Then we call the initAvo Function:
+
+```typescript
+Avo.initAvo({ env: Avo.AvoEnv.Dev }, {}, myAnalyticsSDKBridge);
+```
+
+<br />
+
+Detailed instructions for all other programming languages and platforms supported by Avo [can be found here](/custom-destinations#initialization-instructions).
+
+### Send an event
+
+Now we're ready to start implemening reliably analytics tracking with Avo Functions. Every Event defined in our [Trackin Plan on avo.app](https://www.avo.app/welcome) has it's own functin in the Avo Functions file.
+
+Let's say we have defined the "App Opened" event in our Tracking Plan in Avo, with two Event Properties, "Client" and "Screen Name". Here's how we would send the event using Avo Functions:
+
+```typescript
+Avo.appOpened({ client: 'Web', screenName: 'Home Screen' });
+```
+
+When the Avo Function is called Avo will run validation on the event properties, making sure they fit the spec defined in the Tracking Plan in Avo. Note that this validation is only run in development mode. Avo then passes the event name and properties to the logEvent function in the Custom Destination interface defined above.
+
+You can find tailor made documentation for your Avo Functions, based on your Tracking Plan, in the Implement tab in [your Avo workspace](https://www.avo.app/welcome).
```

Comment:
Should we make a deeplink to `implement/`?

Reply:
With the current routing mechanism we can't, because we don't know you schema id :/

Reply:
We did it for inspector, right?

Reply:
We did 😯? I can't remember, but if we did I also want that here!